src/source/publishing/influx_publisher.cr

```diff
@@ -63,12 +63,12 @@ module PlaceOS::Source
       # Only Module status events are persisted
       return [] of Flux::Point unless data.is_a? Mappings::Status
 
-      payload = message.payload.try &.gsub(Regex.union(DEFAULT_FILTERS)) do |match_string, _|
+      payload = message.payload.presence.try &.gsub(Regex.union(DEFAULT_FILTERS)) do |match_string, _|
         hmac_sha256(match_string)
       end
 
       # Influx doesn't support `nil` values
-      if payload.nil?
+      if payload.nil? || payload == %("null")
```

Comment:
i don't think `null` will be delivered in quotation marks as that would parse as a String

Reply:
I think this is what we're after
```crystal
if payload.nil? || payload == "null"
```